Makefile

```diff
@@ -44,23 +50,25 @@ $(APP_NAME): $(TARGET)
 	@cp -fp $(APP_BINARY) $(APP_BINARY_DIR)
 	@cp -fp $(COMPLETIONS) $(APP_COMPLETIONS_DIR)
 	@touch -r "$(APP_BINARY)" "$(APP_DIR)/$(APP_NAME)"
-	@echo "Created '$@' in '$(APP_DIR)'"
+	@echo "Created '$(APP_NAME)' in '$(APP_DIR)'"
 
-dmg: | $(DMG_NAME) ## Pack Alacritty.app into .dmg
-$(DMG_NAME): $(APP_NAME)
+dmg: | $(DMG_NAME)-native ## Pack Alacritty.app into .dmg
+$(DMG_NAME)-%: $(APP_NAME)-%
 	@echo "Packing disk image..."
 	@ln -sf /Applications $(DMG_DIR)/Applications
 	@hdiutil create $(DMG_DIR)/$(DMG_NAME) \
 		-volname "Alacritty" \
 		-fs HFS+ \
 		-srcfolder $(APP_DIR) \
 		-ov -format UDZO
-	@echo "Packed '$@' in '$(APP_DIR)'"
+	@echo "Packed '$(DMG_NAME)' in '$(APP_DIR)'"
+
+universal: $(DMG_NAME)-universal ## Build universal Alacritty.app and package into .dmg
 
 install: $(DMG_NAME) ## Mount disk image
 	@open $(DMG_DIR)/$(DMG_NAME)
 
-.PHONY: app binary clean dmg install $(TARGET)
+.PHONY: app binary clean dmg install $(TARGET)-native $(TARGET)-universal
 
 clean: ## Remove all artifacts
-	-rm -rf $(APP_DIR)
+	-rm -rf target
```

Comment:
We should probably just run `cargo clean` rather than nuking the target directory. That also destroys everything inside it but it correctly follows overrides to the target directory. It might be located somewhere else.